docs/application/web/api/4.0/ui_fw_api/Mobile_UIComponents/mobile_Drawer.htm

```diff
@@ -316,9 +333,9 @@ <h3>Summary</h3>
 
 					</div>
 					<div class="example">
-						<span class="example"><p>Code
-								example:</p><p></p></span>
-						<pre name="code" class="examplecode
+						<span class="example"><p>Code example:</p><p></p></span>
```

Comment:
In case of another htm file (like mobile_Button.htm) , it also add  the <p></p> "in Code example:"
How about keeping it as it is for the same as other files?

Reply:
As it is a minor issue, I think we can go with this.